src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -117,13 +122,33 @@ export class QuestionSubmissionFormComponent implements DoCheck {
   }
 
   /**
-   * Sorts recipients of feedback by their name.
+   * Compares FeedbackResponseRecipients' names using localeCompare.
+   */
+  private compareByName(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+    return firstRecipient.recipientName.localeCompare(secondRecipient.recipientName);
+  }
+
+  /**
+   * Compares FeedbackResponseRecipients' sections using localeCompare.
    */
-  private sortRecipientsByName(): void {
-    this.model.recipientList.sort((firstRecipient: FeedbackResponseRecipient,
-      secondRecipient: FeedbackResponseRecipient) =>
-      firstRecipient.recipientName.localeCompare(secondRecipient.recipientName));
+  private compareBySection(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+    return firstRecipient!.recipientSection!.localeCompare(secondRecipient.recipientSection!);
```

Comment:
I don't think you need `!` for `firstRecipient`.
Also, it doesn't seem safe to do the non-null assertion even for `recipientSection`. What if it is really null? Similarly for `recipientTeam` below.

Reply:
Understood, I will use a `if` block instead in `recipeintTeam` and `recipeintSection`